src/content/documentation/develop/web-ext-command-reference.md

```diff
@@ -161,6 +161,17 @@ Network port to use when connecting to an Android device with [ADB (Android Devi
 
 Environment variable: `$WEB_EXT_ADB_PORT`
 
+#### \--adb-remove-old-artifacts
+
+`web-ext` does remove automatically all the temporary files that are being pushed to the target adb device when it does exit. This may fail, for example when the device is disconnected before 
```

Comment:
`does remove automatically` -> `automatically removes`

`all the temporary files that are being pushed to the target adb device`. I used "temporary files that were written to target device" opposed to "pushed" because the Firefox app may also write files to the profile (while we still have support for profiles).